code/datums/components/riding/riding_mob.dm

```diff
@@ -157,7 +157,7 @@
 		return
 
 	for(var/mob/yeet_mob in user.buckled_mobs)
-		force_dismount(yeet_mob, (!user.combat_mode)) // gentle on help, byeeee if not
+		force_dismount(yeet_mob, gentle=!user.combat_mode) // gentle on help, byeeee if not
```

Comment:
https://github.com/tgstation/tgstation/blob/c5b7b43fd8eea7e8d7f3baa668ccf977a00c693f/code/datums/components/riding/riding_mob.dm#L301 doesn't have the throw_range, throw_speed and gentle args.

Reply:
Added